_includes/tutorials/fk-joins/kstreams/markup/dev/make-avro-dir.adoc

```diff
@@ -0,0 +1,9 @@
+This tutorial uses three streams: one called `albums` that holds album reference data, one called `trackPurchases` that holds an update-stream of inbound music track purchases, and one called `musicInterestTable` that holds the result of a foreign-key join between purchases and albums.  In the case the inbound keys are different, but the `trackPurcases` stream has the id of the album in its value, so we can use the `KTable` foreign-key join funcitionality to extract the album id and perform the join.
```

Comment:
typo in `trackPurcases`

Reply:
> result of a foreign-key join between purchases and albums
Should this be more explicit and say... 'between trackPurchases and albums'?

Reply:
ack